cli/src/main/java/com/devonfw/tools/ide/tool/LocalToolCommandlet.java

```diff
@@ -175,6 +176,7 @@ private boolean isInstalledVersion(VersionIdentifier expectedVersion, VersionIde
       if (silent) {
         level = IdeLogLevel.DEBUG;
       }
+      // TODO this prints twice since installInRepo also prints this message
```

Comment:
Maybe add an issue for that and link it in the TODO too.

Reply:
Okey, I will do so